corehq/apps/es/case_search.py

```diff
@@ -202,7 +192,7 @@ def case_property_range_query(case_property_name, gt=None, gte=None, lt=None, lt
             case_property_name,
             queries.range_query("{}.{}.numeric".format(CASE_PROPERTIES_PATH, VALUE), **kwargs)
         )
-    except ValueError:
+    except (ValueError, TypeError):
         pass
```

Comment:
There might have been something else. I am not able to reproduce that error any long, so I will just remove that error.

Reply:
You can't remove this. It controls what kind of range query is returned by this function. 
e.g. if you enter a date, it will raise a `ValueError` then pass to the next thing.
You could switch this to an if/else if you think that will be clearer.

Reply:
This does what @esoergel suggests - it checks if the passed in value is a float or a date. This could probably have been implemented differently, but the `try/except` is functioning to check the type.